### ISSUE TITLE
Update PD/PPD link for FT

### ISSUE THREAD

**Issue by JerPearson** [bcgov/SIMS#3971]
PD/PPD links changed to address PT only due to urgency.  Ticket to update content to reflect both FT/PT X8 process.

**Additional Context**
- enter text here
- enter text here

**Acceptance Criteria**
- [ ] Given (Context), When (action carried out), Then (expected outcome)
- [ ] Given (Context), When (action carried out), Then (expected outcome)

**Comment by CarlyCotton** [comment]
@astridSABC Can you add this to hodge podge please?

**Comment by CarlyCotton** [comment]
Bringing out of "Full Full" for content update - need to wait on updates to the "PT" appendix 8.

**Comment by CarlyCotton** [comment]
Deprioritized until cutover from SFAS to SIMS and maybe just a rename required as forms are practically the same.

**Comment by CarlyCotton** [comment]
Closing - another ticket will address the link and PD approach is different.
